src/lib/swapper/swappers/ThorchainSwapper/utils/getL1quote.ts

```diff
@@ -221,14 +222,24 @@ export const getL1quote = async (
             const buyAmountBeforeFeesCryptoBaseUnit =
               getRouteBuyAmountBeforeFeesCryptoBaseUnit(quote)
 
-            const updatedMemo = addSlippageToMemo({
+            const limitWithManualSlippage = getLimitWithManualSlippage({
               expectedAmountOutThorBaseUnit,
-              quotedMemo: quote.memo,
               slippageBps,
-              chainId: sellAsset.chainId,
-              affiliateBps,
-              isStreaming,
             })
+
+            if (!quote.memo) throw new Error('no memo provided')
```

Comment:
suggestion (non-blocking): move this guy at the very beginning of this function (same for the l.333 and l.448 occurences)

Reply:
Done in [ad907d8](https://github.com/shapeshift/web/pull/7138/commits/ad907d8b9e045937eefbd18632d551a76a6b7819)